### PR TITLE
Split the logging into info, warning and error messages

### PR DESCRIPTION
**Pull request by Amey-D**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -70,15 +88,15 @@ configure_kernel_module_locking() {
...
     echo b > /proc/sysrq-trigger
   fi
 }
 
 check_cached_version() {
-  echo "Checking cached version"
+  info "Checking cached version"
   if [[ ! -f "${CACHE_FILE}" ]]; then
-    echo "Cache file ${CACHE_FILE} not found."
+    warn "Cache file ${CACHE_FILE} not found."
```
Comment (rohitagarwal003) [inline entrypoint.sh]: Should this be `warn`?

**Review by rohitagarwal003 — APPROVED** [review]
LGTM except for one minor comment.

File: entrypoint.sh (hunk 1 above)
Comment (Amey-D) [reply entrypoint.sh]: On a second thought this could just be an `info` log.  Is that what you had in mind?

Comment (rohitagarwal003) [reply entrypoint.sh]: Yes.

Comment (Amey-D) [reply entrypoint.sh]: Done

**Pull request merged**